std_msgs/Header header

bool   valid
int32  size

float32 mean_x
float32 mean_y
float32 var_x
float32 var_y

float32 min_x
float32 max_x
float32 min_y
float32 max_y
